extra_time_application/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com)
+# Elico Proprietary License v1.0.
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Extra Time Application',
+    'version': '10.0.0.0.0',
+    'author': "Elico Corp",
+    'website': 'https://www.elico-corp.com',
+    'license': 'Other proprietary',
```

Comment:
You are not being logical/consistent: here it says proprietary but everywhere you say AGPL!
license should be AGPLv3